thirdparty_builtin/gbenchmark-master-2017-05-19/CMakeLists.txt

```diff
@@ -194,9 +194,9 @@ find_package(Threads REQUIRED)
 include_directories(${PROJECT_SOURCE_DIR}/include)
 
 # Build the targets
-add_subdirectory(src)
+add_subdirectory(src ${BLT_BUILD_DIR}/thirdparty_builtin/gbenchmark-master-2017-05-19/src)
```

Comment:
This is a file that is part of the gbenchmark repo.

Is there any way to avoid modifying the CMakeLists file, so we don't have to patch up gbenchmark when we upgrade?  E.g. since you already added the ``add_subdirectory`` call in the parent directory, is it possible that these two calls will work without the second argument?